nixos/lib/make-zfs-image.nix

```diff
@@ -0,0 +1,340 @@
+{ lib
```

Comment:
Can we add a documentation note that this image builder is part of a private API for use internal to Nixpkgs, and can't be relied on for external uses?